admob/src_java/com/google/firebase/admob/internal/cpp/BannerViewHelper.java

```diff
@@ -177,7 +177,6 @@ public void destroy(final long callbackDataPtr) {
     completeBannerViewFutureCallback(callbackDataPtr,
         ConstantsHelper.CALLBACK_ERROR_NONE,
         ConstantsHelper.CALLBACK_ERROR_MESSAGE_NONE);
-    mNotifyBoundingBoxListenerOnNextDraw.set(true);
```

Comment:
This wouldn't have done anything, anyway, as the mAdView was already set to null.  Also there's a synchronous bounding box changed event immediately prior to this.